.github/DISCUSSION_TEMPLATE/ideas.yml

```diff
@@ -0,0 +1,43 @@
+title: "(packageName) Summary of feature or enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for wishing to improve packages that we host in this repository.
+
+        Before you fill out the template here, please make sure that the title contains a very small summary of the the new enhancement or feature, that the title starts with `(packageName)`, and that can be give an idea what the discussion is about.
+        **NOTE: Replace `packageName` with the name of the package**
```

Comment:
Perhaps instead of this note we could include an example of a good title?

Reply:
I have changed this Note to be an example